src/fides/cli/utils.py

```diff
@@ -330,6 +331,7 @@ def handle_aws_credentials_options(
         aws_config = AWSConfig(
             aws_access_key_id=access_key_id,
             aws_secret_access_key=secret_access_key,
+            aws_session_token=session_token,
```

Comment:
interesting, so this seems to still work even when passed through as `None` - seems like the `boto3` client instantiation just ignores any kwargs with `None` values in its constructor, as that's what is going to happen downstream at https://github.com/ethyca/fides/blob/main/src/fides/connectors/aws.py#L23-L33 
```python
service_client = boto3.client(
        service,
        **config_dict,
    )
``` 


nothing to change here since it all works as we want and `boto3` is pretty defensive, just thought it was noteworthy behavior!

Reply:
Yeah I was watching for this to have some issues as I've experienced that with a few other AWS SDKs but it seems that it does a pretty good job at checking here.